### ISSUE TITLE
Bowser does not work on iOS 7 + iPhone 5

### ISSUE THREAD

**Issue by rorya** [EricssonResearch/openwebrtc#526]
Bowser installs on iOS 7, but will not show its UI when launched. The app appears to be running when launched, as seen in the app switcher (2 home button presses) but when bringing it to the foreground, the UI flashes for a second or two before disappearing again.

**Comment by stefanalund** [comment]
Are you installing through the App Store or Xcode?

**Comment by rorya** [comment]
Tried installing it through the App Store. 

iOS version: 7.1.2 (latest v7 version)
Bowser version: 0.5

But yeah, I just tried it again. It launches, the UI shows up for a second and it goes back to the home screen. Even though that's what apps usually do when they crash, I assume it actually is still running because I see it shown it the app switcher. Switching to it that way, does the same thing, the UI is restored for a second and then back to the home screen.

Hope that helps.
